video/out/opengl/osd.c

```diff
@@ -70,14 +71,15 @@ struct mpgl_osd {
     const struct ra_format *fmt_table[SUBBITMAP_COUNT];
     bool formats[SUBBITMAP_COUNT];
     bool change_flag; // for reporting to API user only
+    bool want_pbo;
     // temporary
     int stereo_mode;
     struct mp_osd_res osd_res;
     void *scratch;
 };
 
 struct mpgl_osd *mpgl_osd_init(struct ra *ra, struct mp_log *log,
-                               struct osd_state *osd)
+                               struct osd_state *osd, bool want_pbo)
```

Comment:
Where did you plumb it? Also we call mpgl_osd_init in reinit_from_options anyway.